index1.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <meta charset="utf-8">
+    <title>Project: Rock, Paper, Scissors.</title>
+  </head>
+  <body>
+    <script>
+
+      function computerPlay() {
+        let randomDecimal = Math.random();
```

Comment:
Use `let` if you plan to reassign a variable, if the value should never change after the initial value, then use `const`